x-pack/plugins/triggers_actions_ui/public/common/truncated_text.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+// eslint-disable-next-line @kbn/eslint/module_migration
+import styled from 'styled-components';
+
+const LINE_CLAMP = 2;
+
+const Text = styled.span`
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: ${LINE_CLAMP};
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  word-break: break-word;
+`;
+
+const TruncatedTextComponent: React.FC<{ text: string }> = ({ text }) => <Text>{text}</Text>;
```

Comment:
how the user can see the truncated text, it might be nice to something like that[https://codepen.io/Paulie-D/pen/jxGwdq]( https://codepen.io/Paulie-D/pen/jxGwdq)

Reply:
@XavierM I did not get it. 
I got this solution from cases. It keeps two lines and truncates a text:
<img width="347" alt="Screenshot 2023-03-07 at 09 51 04" src="https://user-images.githubusercontent.com/26089545/223404113-10a0e7ab-5fc3-4ff8-abc9-d7703535b171.png">

Not sure which part of your solution I need to look at and why it's better. Can you tell more?

Reply:
like in the code sand box, if you go over the three dots, it will show the rest of the truncated text.

Reply:
I did not realize that @guskovaue already did something to remediate the problem

<img width="762" alt="image" src="https://user-images.githubusercontent.com/189600/223447926-a62fd911-1977-4564-ba91-b0bd000562c1.png">